src/LibCURL.jl

```diff
@@ -21,27 +14,63 @@ function __init__()
     global cacert = MozillaCACerts_jll.cacert
 end
 
-include("lC_exports_h.jl")
-include("lC_common_h.jl")
-include("lC_curl_h.jl")
+@static if Sys.islinux() && Sys.ARCH === :aarch64 && !occursin("musl", Base.BUILD_TRIPLET)
```

Comment:
Using `@static` at the top level is unnecessary
```suggestion
if Sys.islinux() && Sys.ARCH === :aarch64 && !occursin("musl", Base.BUILD_TRIPLET)
```
I would also define a constant for `occursin("musl", Base.BUILD_TRIPLET)`